Networking/WebSockets/WebSocketProtocol.hh

```diff
@@ -160,18 +167,34 @@ private:
     inline bool consumeMessage(T payLength, char *&src, unsigned int &length, frameFormat frame, void *user) {
         if (getOpCode(frame)) {
             if (opStack == 1 || (!lastFin && getOpCode(frame) < 2)) {
-                forceClose(user, "cm-1");
+#ifdef COUCHBASE_forceClose
+                std::stringstream ss;
+                ss << "[opStack=1,lastFin=" << lastFin << ",frame=" <<frame << "]";
```

Comment:
* output opStack instead of 1 as it may not be one.
* (format) Add a space between << and frame.

Reply:
I'm not sure what will be printed as `frame`, at least `opcode` should be printed.